libs/blocks/global-navigation/global-navigation.css

```diff
@@ -628,6 +639,14 @@ header.global-navigation {
     fill: var(--feds-color-link--hover--light);
   }
 
+  .feds-utilities {
+    padding: 0;
+  }
+
+  [dir = "rtl"] .feds-utilities {
+    padding: 0;
+  }
```

Comment:
You could simply group these together
```
.feds-utilities,
[dir = "rtl"] .feds-utilities {
  padding: 0;
}
```

LE: We should probably have some padding, though, as the focus state of the right side logo overlaps the utilities:
<img width="300" alt="Screenshot 2024-01-05 at 12 17 51" src="https://github.com/adobecom/milo/assets/11267498/82bc38b6-d203-4137-aee9-e64e80f62919">